### DISCUSSION TITLE
[FR]: Option to remove tags after adding them to frontmatter

### DISCUSSION THREAD

**Discussion by ennui2342** [ObsidianPublisher/obsidian-github-publisher#62] [Q&A — Plugin] [answered]
### Issue validation

- [X] My issue is not a question relative to [Material Mkdocs](https://squidfunk.github.io/mkdocs-material/) nor [Mkdocs](https://www.mkdocs.org/) (*[You can open a discussion to get more help!](https://github.com/squidfunk/mkdocs-material/discussions)*)
- [X] I checked the issue to prevent duplicate

### Workflow

Obsidian plugin 💠

### What solution do you want to see ?

When I choose the option to move tags into frontmatter so that they're picked up in Jekyll tag management, this all works perfectly, but the tags remain also inline in the published document. This ends up looking something like this in my Jekyll blog:

<img width="379" alt="image" src="https://user-images.githubusercontent.com/912310/209433879-3cdf363d-24d2-4f75-946b-2a5c2e06d57f.png">

I'd like an additional option for the tags to be removed from the document when they're added to the frontmatter. Thanks.

-Mark

### OS

MacOS

**Comment by Mara-Li** [comment, marked as answer]
Hello,
Normally, the text replacement using the option to use it AFTER all conversion will do that. 

![image](https://user-images.githubusercontent.com/30244939/209438613-60fade5e-54ea-4fe9-a54f-171029887763.png)

Configuration for that : 

![image](https://user-images.githubusercontent.com/30244939/209438628-7ddac170-8ab9-4e19-aea0-5de99f55870d.png)

(I put a space for the replacement)

**Comment by ennui2342** [comment]
Yep, thanks for this. You're absolutely right this can be used - hadn't thought of that method. To make it a bit more robust I'd suggest the regular expression should be #[^ ]+

Thanks for the quick reply!
